From abad2b770a0fe28b51ac07827be7a80c4bb32475 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 29 Jul 2020 09:12:25 -0700
Subject: [PATCH 1/3] Disable syscalls for Emscripten

Fixes error:

    src/raw_logging.cc:139:3: error: use of undeclared identifier 'syscall'
      safe_write(STDERR_FILENO, buffer, strlen(buffer));
      ^
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 808330e..d1cb98a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,11 +72,9 @@ check_include_file (stdlib.h HAVE_STDLIB_H)
 check_include_file (string.h HAVE_STRING_H)
 check_include_file (strings.h HAVE_STRINGS_H)
 check_include_file (sys/stat.h HAVE_SYS_STAT_H)
-check_include_file (sys/syscall.h HAVE_SYS_SYSCALL_H)
 check_include_file (sys/time.h HAVE_SYS_TIME_H)
 check_include_file (sys/types.h HAVE_SYS_TYPES_H)
 check_include_file (sys/utsname.h HAVE_SYS_UTSNAME_H)
-check_include_file (syscall.h HAVE_SYSCALL_H)
 check_include_file (syslog.h HAVE_SYSLOG_H)
 check_include_file (ucontext.h HAVE_UCONTEXT_H)
 check_include_file (unistd.h HAVE_UNISTD_H)
-- 
2.17.1

